tests/Runtimes/RuntimesTest.php

```diff
@@ -131,29 +131,23 @@ public function setUp(): void
                 'timeout' => 15,
                 'runtime' => 'dotnet-5.0'
             ],
-            'java-11' => [
-                'code' => $functionsDir . '/java-11.tar.gz',
+            'java-8.0' => [
+                'code' => $functionsDir . '/java-8.tar.gz',
                 'command' => 'java HelloWorld',
                 'timeout' => 15,
-                'runtime' => 'java-11'
-            ],
-            'kotlin-jar-11' => [
-                'code' => $functionsDir . '/kotlin.tar.gz',
-                'command' => 'java -jar HelloWorld.jar',
-                'timeout' => 15,
-                'runtime' => 'java-11'
+                'runtime' => 'java-8.0'
             ],
-            'java-16' => [
+            'java-11.0' => [
                 'code' => $functionsDir . '/java-11.tar.gz',
                 'command' => 'java HelloWorld',
                 'timeout' => 15,
-                'runtime' => 'java-16'
+                'runtime' => 'java-11.0'
             ],
-            'kotlin-jar-16' => [
-                'code' => $functionsDir . '/kotlin.tar.gz',
-                'command' => 'java -jar HelloWorld.jar',
+            'java-17.0' => [
+                'code' => $functionsDir . '/java-17.tar.gz',
+                'command' => 'java HelloWorld',
                 'timeout' => 15,
-                'runtime' => 'java-16'
+                'runtime' => 'java-17.0'
             ],
```

Comment:
IIRC, we also released new Deno version.. Maybe even others? Can we please double-check?

Reply:
I don't think deno-1.20 will be merged in time for this release. There are some failing tests in there